src/commands/containers/selectContainer.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { ContainerTreeItem } from '../../tree/containers/ContainerTreeItem';
+
+export async function selectContainer(context: IActionContext): Promise<string> {
+
+    let node: ContainerTreeItem;
+
+    //Expecting running containers to change often as this is a debugging scenario.
+    await ext.containersTree.refresh();
+
+    node = await ext.containersTree.showTreeItemPicker(/^(running)Container$/i, { ...context });
```

Comment:
Why the parens in the regex?  It's same as matching runningContainer.

Reply:
Copied and altered from other command, I'll fix tonight.

Reply:
Yeah parens are only needed if multiple different states are allowed - like `/^(running|paused)Container$/i`

Reply:
Resolved!